src/cljs/org/broadinstitute/firecloud_ui/page/profile.cljs

```diff
@@ -35,14 +35,7 @@
 
 
 (react/defc Page
-  {:get-initial-state
-   (fn [{:keys [props]}]
-     (when (:new-registration? props)
-       (let [google-profile (-> (utils/get-current-user)
-                                (utils/call-external-object-method :getBasicProfile))]
-         {:initial-values {:name (utils/call-external-object-method google-profile :getName)
```

Comment:
did you experiment at all with using the google api, and manually init-ing it with the access token? (I don't know if this is possible, or if maybe you need the id token too, or what). Again, nothing to change for this PR, but I think that is an interesting line of research, to see if we can get the user's name etc. back.